lib/jsonStore.ts

```diff
@@ -11,15 +12,15 @@ const logger = module('Store')
 /**
 Constructor
 **/
-class StorageHelper {
-	public store: any
-	public config: any
+export class StorageHelper {
+	public store: Record<StoreKeys, any>
+	public config: Record<StoreKeys, StoreFile>
 
 	constructor() {
-		this.store = {}
+		this.store = {} as Record<StoreKeys, any>
```

Comment:
This shouldn't be necessary when you define the type next to the property.

Reply:
```
Type '{}' is missing the following properties from type 'Record<StoreKeys, any>': settings, scenes, nodes, users
```

Whithout that I got this error